requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
It is not a good idea to generate in a python 3.11 venv. It could potentially break things. We have to be aware, that we do not break dependencies for our lowest supported python version 3.9

Reply:
That's true, haven't noticed that. I have redone it.